generators/php/lists.js

```diff
@@ -356,6 +356,34 @@ Blockly.PHP['lists_getSublist'] = function(block) {
   return [code, Blockly.PHP.ORDER_FUNCTION_CALL];
 };
 
+Blockly.PHP['lists_sort'] = function(block) {
+  // Block for sorting a list.
+  var listCode = Blockly.PHP.valueToCode(block, 'LIST',
+      Blockly.PHP.ORDER_FUNCTION_CALL) || 'array()';
+  var direction = block.getFieldValue('DIRECTION') === '1' ? 1 : -1;
+  var type = block.getFieldValue('TYPE');
+  var functionName = Blockly.PHP.provideFunction_(
+    'lists_sort', [
+    'function ' + Blockly.PHP.FUNCTION_NAME_PLACEHOLDER_ + 
+      '($list, $type, $direction) {',
+    '  $sortFuncs = array(',
+    '    "numeric" => "sort",',
+    '    "text" => "natsort",',
+    '    "ignoreCase" => "natcasesort"',
+    '  );',
+    '  $sort = $sortFuncs[$type];',
+    '  $list2 = $list;', // clone list
```

Comment:
We prefer that comments start with uppercase and end with punctuation.  As done in the Dart generator.